_posts/newsletters/2023-03-29-newsletter.md

```diff
@@ -0,0 +1,282 @@
+<br>
+<br>
+To keep this podcast as a true community effort it is important to hear from you! I look forward to hear about your ideas, comments and suggestions on <a href="mailto:code4thought@proton.me">email</a>, <a href="https://www.linkedin.com/company/codeforthought/">LinkedIn</a>, <a href="https://ukrse.slack.com/join/signup#/domain-signup">UK Slack workspace</a> (@code4thought), <a href="https://fosstodon.org/@code4thought">Mastodon</a>, and <a href="https://code4thought-podcast.github.io/newsletters/">Github news/blogs</a>. 
+<br>
+<br>
+My heartfelt thanks for listening and supporting this podcast. I wish you all the best for 2023 and I hope you enjoy the show.
+<br>
+<br>
+Peter
+</blockquote>
+
+Items you may have missed on the blog and Slack:
+    
+  * Blogs and podcasts with US-RSE members
+     * [Defining the roles of research software](https://doi.org/10.54900/9akm9y5-5ject5y) by Rob van Nieuwpoort and Daniel S. Katz
+     * [Developer Stories Podcast](https://rseng.github.io/devstories/posts/)
+  * Blogs and podcasts from beyond US-RSE
+     * [Code for Thought Podcasts](https://codeforthought.buzzsprout.com)
+     * [Sustaining Open Source Ecosystems Podcasts](https://podcast.sustainoss.org)
+
+* The organizers of RSECon2022 have released a playlist with 53 videos so far 
+    * <a name="https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj"></a>[RSECon2022 Playlist](https://www.youtube.com/playlist?list=PL27mQJy8eDHk---gHCapHwyVaWIkyORYj)
+
+* The latest [Research Software Alliance (ReSA) newsletter](https://preview.mailerlite.com/p3p4u2g7h9/2175082983801755815/u9c5/) includes a lot of research software community news and events.
+
+<a name="outreach"></a>
+# **US-RSE Outreach**
+
+The Outreach Working Group is looking for volunteers to help with organizing workshops, birds of a feather (BoF), symposiums, etc. Roles range from organizer to reviewer to speaker and more.
+
+If you are interested in getting involved in a US-RSE hosted event, please reach out to the working group via the [#wg-outreach](https://app.slack.com/client/T8ZT4PJSW/CNR6E0N75) slack channel or [wg-outreach@us-rse.org](mailto:wg-outreach@us-rse.org).
+
+<a name="financials"></a>
+# **US-RSE Has a New Fiscal Sponsor!**
+
+We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.
```

Comment:
```suggestion
We’re excited to announce that US-RSE has switched to a new fiscal sponsor. We are now a fiscally-sponsored project of [Community Initiatives](http://communityin.org/) (CI)! Our previous fiscal sponsor, Open Collective Foundation (OCF), represented an important first step allowing us to accept donations and making it possible to reimburse members for expenses they incurred on behalf of US-RSE. Moving forward, Community Initiatives’ additional tools, resources, and infrastructure will better serve US-RSE’s future organizational and financial goals.
```

Reply:
I'd disagree with this one given the `-ly` suffix of the adverb.

Reply:
to me, "fiscally-sponsored" is an adjective phrase that should be hyphenated. It's a question of style, however.